### ISSUE TITLE
After upgrade the raspberry pi 4B I got wrong webserver

### ISSUE THREAD

**Issue by LightBridge16** [bassmaster187/TeslaLogger#1369]
I have made a mistake or wrong update/upgrade proces?😬😅
![IMG_0386](https://github.com/user-attachments/assets/7ac31f67-d64b-4b62-9fca-e601f01657a1)

Please can someone help me as fast as possible 🙏🏻🙏🏻🙏🏻

**Comment by bassmaster187** [comment]
Did you make any changes on apache or php?

**Comment by LightBridge16** [comment]
There was ask that 3 difference by upgrading php and I keeps that 3 instead upgrade it with the new settings. Op 9 okt. 2024 om 08:32 heeft Christian P. ***@***.***> het volgende geschreven:﻿
Did you make any changes on apache or php?

**Comment by LightBridge16** [comment]
Can I do a new up/grade for pho that will fix the “missing” links?

**Comment by bassmaster187** [comment]
so you did a apt-get update & apt-get upgrade and you were asked if you wanted to overwrite some files?

**Comment by LightBridge16** [comment]
Yes and I didn’t 😬Op 12 okt. 2024 om 22:10 heeft Christian P. ***@***.***> het volgende geschreven:﻿
so you did a apt-get update & apt-get upgrade and you were asked if you wanted to overwrite some files?

**Comment by LightBridge16** [comment]
Uh I upgraded too bullseye

**Comment by LightBridge16** [comment]
got this messages:

The following packages have unmet dependencies:
 php-mysqlnd-ms : Depends: php8.3-mysqlnd-ms but it is not installable
 php5.6-dev : Depends: autoconf (>= 2.63) but it is not installable
              Depends: automake (>= 1.11) but it is not installable
              Depends: libpcre3-dev (>= 2:8.35) but it is not going to be installed
              Depends: shtool but it is not installable
              Depends: libtool (>= 2.4.6-0.1~) but it is not installable
              Recommends: pkg-php-tools but it is not installable
 php5.6-xml : Depends: libxslt1.1 (>= 1.1.25) but it is not installable
 php7.0-dev : Depends: autoconf (>= 2.63) but it is not installable
              Depends: automake (>= 1.11) but it is not installable
              Depends: libpcre3-dev (>= 2:8.35) but it is not going to be installed
              Depends: shtool but it is not installable
              Depends: libtool (>= 2.4.6-0.1~) but it is not installable
              Recommends: pkg-php-tools but it is not installable
 php7.0-xml : Depends: libxslt1.1 (>= 1.1.25) but it is not installable
 php7.1-dev : Depends: autoconf (>= 2.63) but it is not installable
              Depends: automake (>= 1.11) but it is not installable
              Depends: libpcre3-dev (>= 2:8.35) but it is not going to be installed
              Depends: shtool but it is not installable
              Depends: libtool (>= 2.4.6-0.1~) but it is not installable
              Recommends: pkg-php-tools but it is not installable
 php7.1-xml : Depends: libxslt1.1 (>= 1.1.25) but it is not installable
 php7.2-dev : Depends: autoconf (>= 2.63) but it is not installable
              Depends: automake (>= 1.11) but it is not installable
              Depends: libpcre3-dev (>= 2:8.35) but it is not going to be installed
              Depends: shtool but it is not installable
              Depends: libtool (>= 2.4.6-0.1~) but it is not installable
              Recommends: pkg-php-tools but it is not installable
 php7.2-xml : Depends: libxslt1.1 (>= 1.1.25) but it is not installable
 php7.3-dev : Depends: autoconf (>= 2.63) but it is not installable
              Depends: automake (>= 1.11) but it is not installable
              Depends: libpcre2-dev (>= 10.30) but it is not going to be installed
              Depends: shtool but it is not installable
              Depends: libtool (>= 2.4.6-0.1~) but it is not installable
              Recommends: pkg-php-tools but it is not installable
 php7.3-xml : Depends: libxslt1.1 (>= 1.1.25) but it is not installable
 php7.4-dev : Depends: autoconf (>= 2.63) but it is not installable
              Depends: automake (>= 1.11) but it is not installable
              Depends: libpcre2-dev (>= 10.30) but it is not going to be installed
              Depends: shtool but it is not installable
              Depends: libtool (>= 2.4.6-0.1~) but it is not installable
              Recommends: pkg-php-tools but it is not installable
 php7.4-mbstring : Depends: libonig5 (>= 6.8.1) but it is not installable
 php7.4-xml : Depends: libxslt1.1 (>= 1.1.25) but it is not installable
 php8.0-dev : Depends: autoconf (>= 2.63) but it is not installable
              Depends: automake (>= 1.11) but it is not installable
              Depends: libpcre2-dev (>= 10.30) but it is not going to be installed
              Depends: shtool but it is not installable
              Depends: libtool (>= 2.4.6-0.1~) but it is not installable
              Recommends: pkg-php-tools but it is not installable
 php8.0-mbstring : Depends: libonig5 (>= 6.8.1) but it is not installable
 php8.0-xml : Depends: libxslt1.1 (>= 1.1.25) but it is not installable
 php8.1-dev : Depends: autoconf (>= 2.63) but it is not installable
              Depends: automake (>= 1.11) but it is not installable
              Depends: libpcre2-dev (>= 10.30) but it is not going to be installed
              Depends: shtool but it is not installable
              Depends: libtool (>= 2.4.6-0.1~) but it is not installable
              Recommends: pkg-php-tools but it is not installable
 php8.1-mbstring : Depends: libonig5 (>= 6.8.1) but it is not installable
 php8.1-xml : Depends: libxslt1.1 (>= 1.1.25) but it is not installable
 php8.2-dev : Depends: autoconf (>= 2.63) but it is not installable
              Depends: automake (>= 1.11) but it is not installable
              Depends: libpcre2-dev (>= 10.30) but it is not going to be installed
              Depends: shtool but it is not installable
              Depends: libtool (>= 2.4.6-0.1~) but it is not installable
              Recommends: pkg-php-tools but it is not installable
 php8.2-mbstring : Depends: libonig5 (>= 6.8.1) but it is not installable
 php8.2-xml : Depends: libxslt1.1 (>= 1.1.25) but it is not installable
 php8.3-dev : Depends: autoconf (>= 2.63) but it is not installable
              Depends: automake (>= 1.11) but it is not installable
              Depends: libpcre2-dev (>= 10.30) but it is not going to be installed
              Depends: shtool but it is not installable
              Depends: libtool (>= 2.4.6-0.1~) but it is not installable
              Recommends: pkg-php-tools but it is not installable
 php8.3-mbstring : Depends: libonig5 (>= 6.8.1) but it is not installable
 php8.3-xml : Depends: libxslt1.1 (>= 1.1.25) but it is not installable
 php8.4-dev : Depends: autoconf (>= 2.63) but it is not installable
              Depends: automake (>= 1.11) but it is not installable
              Depends: libpcre2-dev (>= 10.30) but it is not going to be installed
              Depends: shtool but it is not installable
              Depends: libtool (>= 2.4.6-0.1~) but it is not installable
              Recommends: pkg-php-tools but it is not installable
 php8.4-mbstring : Depends: libonig5 (>= 6.8.1) but it is not installable
 php8.4-xml : Depends: libxslt1.1 (>= 1.1.27) but it is not installable
E: Unable to correct problems, you have held broken packages.
Considering dependency mpm_prefork for php7.3:
Considering conflict mpm_event for mpm_prefork:
Considering conflict mpm_worker for mpm_prefork:
Module mpm_prefork already enabled
Considering conflict php5 for php7.3:
Enabling module php7.3.
To activate the new configuration, you need to run:
  systemctl restart apache2
Reading datadir from the MariaDB server failed. Got the following error when executing the 'mysql' command line client
ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/run/mysqld/mysqld.sock' (2)
FATAL ERROR: Upgrade failed
PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
NAME="Raspbian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"
ldd (Debian GLIBC 2.36-9+rpt2+deb12u8) 2.36
